addons/sale_stock/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,17 @@
+---Models in module 'sale_stock'---
+---Fields in module 'sale_stock'---
+sale_stock   / account.invoice          / incoterms_id (many2one)       : DEL relation: account.incoterms
+# NOTHING TO DO: This field was duplicated in the v12 DB schema, and it was hidden in views, so no possible values in it
```

Comment:
Now model `account.move` and field `invoice_incoterm_id` so was done in an `account` module

Reply:
Not really. That's another field. An upcoming v12 patch will "merge" the values of both fields where it belongs.

Reply:
See #2544

Reply:
Oh, I see. Thanks.